timetomodel/speccing.py

```diff
@@ -106,11 +111,16 @@ def load_series(
 
            `resampling_config={"closed": "left", "aggregation": "sum"}`
 
+            Similarly, pass in an `interpolation_config` to the class with kw params to pass into `interpolate`.
```

Comment:
I guess here and above (for resample) we should add the fact that we mean `pandas.Series.interpolate`.